source/hardware/controllers/allen_heath_xone_k2_k1.rst

```diff
@@ -295,13 +230,15 @@ manual <https://allen-heath.com/media/Xone+K2_UG_AP8509_2.pdf>`__ page
 Known Issues
 ------------
 
-There are a few known issues with the Xone K2/K1 firmware. First, there
-is no way for Mixxx to ask the controller about the positions of all the
-knobs and faders when Mixxx starts, so they are out of sync with the
-state of Mixxx until they are first moved. Also, when the USB connection
-is interrupted and the controller stays powered on, such as leaving the
-controller plugged into a powered USB hub and disconnecting the hub from
-the computer, the controller will not send MIDI input until it is
-powered down (unplugged) and plugged back in. If you would like these
-issues to be fixed, please let `Allen & Heath
+There are a few known issues with the Xone K2/K1 firmware:
+
+-  Since there is no way for Mixxx to ask the controller about the
+positions of all the knobs and faders when Mixxx starts, they are
+out of sync with the state of Mixxx until they are first moved.
+-  When the USB connection is interrupted and the controller stays
+powered on, such as leaving the controller plugged into a powered
+USB hub and disconnecting the hub from the computer, the controller
+will not send MIDI input until it is powered down (unplugged) and plugged back in.
```

Comment:
The formatting is broken.

```suggestion
-  Since there is no way for Mixxx to ask the controller about the positions of
   all the knobs and faders when Mixxx starts, they are out of sync with the
   state of Mixxx until they are first moved.
-  When the USB connection is interrupted and the controller stays powered on,
   such as leaving the controller plugged into a powered USB hub and
   disconnecting the hub from the computer, the controller will not send MIDI
   input until it is powered down (unplugged) and plugged back in.
```